adoc/deployment-bootstrap.adoc

```diff
@@ -3,13 +3,68 @@
 
 === Cluster Deployment
 
+==== Cluster initialization
 . Initialize the cluster on the deployed machines.
 As `--control-plane` enter the IP/FQDN of your load balancer. If you do not use a load balancer use your first master node.
 +
 ----
 skuba cluster init --control-plane <LB IP/FQDN> my-cluster
 ----
 `cluster init` generates the folder named `my-cluster` and initializes the directory that will hold the configuration for the cluster.
+
+==== Cluster configuration
+
+At this point you can perform some configurations before bootstrapping the cluster.
+
+===== Prevent nodes running special workloads from being rebooted
+
+. Open the `kured` deployment in `my-cluster/addons/kured/kured.yaml`
```

Comment:
I think that it would be good to explicitly mention that if the cluster is already bootstrapped they can `kubectl edit` the daemonset and achieve the same goal.

Reply:
true, i almost forgot. thanks for the hint